Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/SyncIntTests.swift

```diff
@@ -1936,8 +2010,12 @@ class SyncIntTests: BaseStitchIntTestCocoaTouch {
 
         XCTAssertNil(coll.findOne(doc1Filter))
         XCTAssertNil(remoteColl.findOne(doc1Filter))
+
+        coll.verifyUndoCollectionEmpty()
     }
 
+    // MARK: Utilities
```

Comment:
[q] unfamiliar with this notation, can you explain?

Reply:
Just a marker for XCode so that you can quickly jump to this section of the file from the UI:

![image](https://user-images.githubusercontent.com/1976694/50175437-ce577c00-02ca-11e9-85b5-1e083fba3e0b.png)